### PR TITLE
Minor correction

### PR DESCRIPTION
**Pull request by pitipatw**
### Summary

A short summary of the changes in this PR.

**Highlights**:
1. the argument in visualize does not have "u" (in u = solver.u , so it could not show the Makie figure. 
2. I recheck with the VDO by Yijiang https://www.youtube.com/watch?v=sBqdkxPXluU and corrected them as in the clip, and now it works fine.

**Minor changes**:
1. change "; u= solver.u" 
2. to "solver.u"

### What type of change is this?

- [ ] Bug fix in a **backwards-compatible** manner.
- [ ] New feature in a **backwards-compatible** manner.
- [ /] Breaking change: bug fix or new feature that involve incompatible API changes.
- [ ] Other (e.g. doc update, configuration, etc)

### Checklist

- [/ ] I ran all tests on my computer and it's all green (i.e. `] test`).
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by mohamed82008** [comment]
Thanks for the PR @pitipatw! We used to comment out the visualization lines because getting visualization code to run on GitHub actions was a pain. Can you please comment the lines out again after the correction you mentioned?

**Comment by pitipatw** [comment]
Sure! I will do that

File: test/wcsmo14/demos/truss/truss_compliance_2d2.jl [hunk 1]
```diff
@@ -44,10 +44,10 @@ TopOpt.setpenalty!(solver, p)
 
 @show obj(r.minimizer)
 @show constr(r.minimizer)
-# fig = visualize(
-#     problem; u=solver.u, topology = r.minimizer,
-#     default_exagg_scale=0.0
-# )
-# Makie.display(fig)
+fig = visualize(
+    problem, solver.u, topology = r.minimizer,
```
Comment (pitipatw) [inline test/wcsmo14/demos/truss/truss_compliance_2d2.jl:48]: This part. 

The code fails to recognize u as one of the argument, and a semi-colon was used after problem

Remove them would make the script run smoothly.

Comment (mohamed82008) [reply test/wcsmo14/demos/truss/truss_compliance_2d2.jl:48]: @yijiangh is this a recent change?

Comment (yijiangh) [reply test/wcsmo14/demos/truss/truss_compliance_2d2.jl:48]: @pitipatw the visualizer should be able to pick up `u` by calling `fig = visualize(problem; solver.u, topology=minimizer)`. The related source code can be found [here](https://github.com/JuliaTopOpt/TopOpt.jl/blob/5c1c74a9662557b4a4a2f3c8197c7a398fc4d081/src/TopOptProblems/Visualization/makie.jl#L175-L176). Would you mind trying again?

BTW, the youtube video is a bit outdated for our current master branch. Sorry for that.

Comment (pitipatw) [reply test/wcsmo14/demos/truss/truss_compliance_2d2.jl:48]: I tried fig = visualized(problem; solver.u, topology=minimizer) and get error :MethodError: no method matching visualize(...

Currently, I can run fig = visualize(problem , solver.u ... without error

only the comma (,) and semi-colon (;) that is different.

Comment (yijiangh) [reply test/wcsmo14/demos/truss/truss_compliance_2d2.jl:48]: Ah! You are using v0.6.1, and I did clean up the visualization code to make `u` appears after the semicolon `;`, change can be found [here](https://github.com/JuliaTopOpt/TopOpt.jl/compare/v0.6.1...master#diff-c012c87b78f02b909b1b32a8623ad5269a7ac1ef08c502917b7cba4caab53ebeL20-L23).
If you pull the master branch, you should see that problem go away!

@mohamed82008 shall we do a minor version bump to v0.6.2 for the viz?

Comment (mohamed82008) [reply test/wcsmo14/demos/truss/truss_compliance_2d2.jl:48]: let's release 0.7

Comment (pitipatw) [reply test/wcsmo14/demos/truss/truss_compliance_2d2.jl:48]: I see.

**Pull request closed without merging**